SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -196,11 +196,11 @@ float USpatialGDKSettings::GetSecondsBeforeWarning(const ERPCResult Result) cons
 	return RPCQueueWarningDefaultTimeout;
 }
 
-bool USpatialGDKSettings::GetPreventClientCloudDeploymentAutoConnect(bool bIsClient) const
+bool USpatialGDKSettings::GetPreventClientCloudDeploymentAutoConnect() const
 {
-#if WITH_EDITOR
+#if UE_EDITOR || UE_SERVER
 	return false;
 #else
-	return bIsClient && bPreventClientCloudDeploymentAutoConnect;
+	return bPreventClientCloudDeploymentAutoConnect;
```

Comment:
Can we check that `-server` hasn't been passed to this instance also?

Reply:
Sure, in command line args?

Reply:
Yeah - I could have sworn there was an existing unreal function which checks the netdriver first, and if it doesn't exist, checks the command line args for `-server` , but can't find it.